### DISCUSSION TITLE
Where is GitHub Actions in iOS app?

### DISCUSSION THREAD

**Discussion by Justin-JHG** [community/community#36619] [Mobile]
![image](https://user-images.githubusercontent.com/41654878/196601390-8e36972c-d053-40b1-89fc-133a808a2665.jpeg)

Re above release notes GitHub Actions available in iOS app now, but I can't seem to find it ?

**Comment by vykouzlith2fb3b** [comment]
Select the 'Actions' tab on your repository and choose the Simple workflow template.[mobile app development](https://www.applicontech.com/#services). This is the easiest way to start using GitHub actions, as it will create all the essentials that you need to have in a workflow.

**Reply by Justin-JHG** [reply]
thank you @vykouzlith2fb3b 

I have Actions set up in my repo, but I cannot see them in GitHub iOS App

**Comment by candyho** [comment]
Hey @Justin-JHG - you can view your workflow run summaries and logs when you navigation to any Pull Request's detail view, and expand the "Checks" section.

**Comment by Justin-JHG** [comment]
Hi @candyho ,



Sorry my actions are triggered after pull request merged to do the build.

Does that mean I can't view those Actions in iOS app ?

Any plan to add GitHub Actions full feature to the iOS app? It will be very useful

Thanks


Justin

**Comment by ossanna16** [comment]
Hey there @Justin-JHG and thank you for asking a good question! I'm glad you found our community 🙂
